DQMOffline/Trigger/plugins/DiJetMonitor.h

```diff
@@ -0,0 +1,125 @@
+
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/Registry.h"
+
+#include "DQMOffline/Trigger/plugins/TriggerDQMBase.h"
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+#include "CommonTools/Utils/interface/StringCutObjectSelector.h"
+
+//DataFormats
+#include "DataFormats/METReco/interface/PFMET.h"
+#include "DataFormats/METReco/interface/PFMETCollection.h"
+
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/JetReco/interface/PFJetCollection.h"
+#include "DataFormats/JetReco/interface/CaloJet.h"
+#include "DataFormats/JetReco/interface/CaloJetCollection.h"
+#include "DataFormats/JetReco/interface/GenJetCollection.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectronFwd.h"
+#include "DataFormats/EgammaCandidates/interface/Photon.h"
+#include "DataFormats/EgammaCandidates/interface/PhotonFwd.h"
+
+class GenericTriggerEventFlag;
+
+//
+// class declaration
+//
+
+class DiJetMonitor : public DQMEDAnalyzer  ,  public TriggerDQMBase
+{
+public:
+  DiJetMonitor( const edm::ParameterSet& );
+  virtual ~DiJetMonitor() noexcept(true) {};
+  static void fillDescriptions(edm::ConfigurationDescriptions & descriptions);
+
+protected:
+
+  void bookHistograms(DQMStore::IBooker &, edm::Run const &, edm::EventSetup const &) override;
+  void analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup) override;
+  bool dijet_selection(double eta_1, double phi_1, double eta_2, double phi_2, double pt_1, double pt_2, int &tag_id, int &probe_id);
+
+private:
+
+  std::string folderName_;
+  std::string histoSuffix_;
+
+  edm::EDGetTokenT<reco::PFMETCollection>       metToken_;
+  edm::EDGetTokenT<reco::GsfElectronCollection> eleToken_;
+  edm::EDGetTokenT<reco::MuonCollection>        muoToken_;
+  edm::EDGetTokenT<reco::PFJetCollection>  dijetSrc_; // test for Jet
+
+  std::vector<double> dijetpT_variable_binning_;
+  MEbinning           dijetpt_binning_;
+  MEbinning           dijetptThr_binning_;
+  MEbinning           ls_binning_;
+
+  ObjME jetpt1ME_;
+  ObjME jetpt2ME_;
+  ObjME jetptAvgaME_;
+  ObjME jetptAvgaThrME_;
+  ObjME jetptAvgbME_;
+  ObjME jetptTagME_;
+  ObjME jetptPrbME_;
+  ObjME jetptAsyME_;
+  ObjME jetetaPrbME_;
+  ObjME jetetaTagME_;
+  ObjME jetphiPrbME_;
+  ObjME jetAsyEtaME_;
+  ObjME jetEtaPhiME_;
+
+  std::unique_ptr<GenericTriggerEventFlag> num_genTriggerEventFlag_;
+  std::unique_ptr<GenericTriggerEventFlag> den_genTriggerEventFlag_;
+
+  int nmuons_;
+  double ptcut_;
+  double etacut_;
+  double phicut_;
+
+  std::vector<double> v_jetpt;
```

Comment:
Sorry for late replying.
Does your comment mean that etacut_ and phicut_ should be located in code as number?
For example, do I change 'if (abs(eta_1)< etacut_ )' into 'if (abs(eta_1)< 1.7 )' ?